buildspec.yml

```diff
@@ -14,12 +16,13 @@ phases:
     runtime-versions:
       nodejs: 14
     commands:
-      # updating npm to version 8.x.x (it is not done automatically by settings env to nodejs v14)
-      # see package.json for version requirements
-      - npm install -g npm@8.4.1
+      # updating node to v16
+      - n 16
+      # As the cypress/download folder is not added to git, and (somehow) not created by Cypress at startup, we create it
+      - mkdir cypress/downloads/
```

Comment:
`mkdir -p` to create parents if `cypress` dir would not exist might be less brittle.